docroot/modules/custom/annotation_store/src/AnnotationStoreInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Drupal\annotation_store;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\user\EntityOwnerInterface;
+use Drupal\Core\Entity\EntityChangedInterface;
+
+/**
+ * Provides an interface defining a AnnotationStore entity.
+ *
+ * We have this interface so we can join the other interfaces it extends.
+ *
+ * @ingroup annotation_store
+ */
+interface AnnotationStoreInterface extends ContentEntityInterface, EntityOwnerInterface, EntityChangedInterface {
+
```

Comment:
Add definition for custom getter & setter methods defined for custom properties.